### PR TITLE
docs: using scaleway vpc bastion ssh

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: docs/guides/using-vpc-bastion-ssh.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+page_title: "Using Scaleway Bastion SSH"
+description: |-
+  Using Scaleway Bastion SSH config.
+---
+
+# Using Scaleway VPC Bastion SSH config
```
Comment (Vinzius) [inline docs/guides/using-vpc-bastion-ssh.md:7]: Not sure what formulation is best.

```suggestion
# How to use Scaleway VPC Bastion SSH config
```

File: docs/guides/using-vpc-bastion-ssh.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+In this guide you'll learn how to deploy Scaleway Bastion with to your virtual network using the Scaleway Terraform provider.
```
Comment (Vinzius) [inline docs/guides/using-vpc-bastion-ssh.md:9]:
```suggestion
In this guide you'll learn how to deploy Scaleway Bastion to your virtual network using the Scaleway Terraform provider.
```

File: docs/guides/using-vpc-bastion-ssh.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+When you connect to a VM, it doesn't need a public IP address, client software, agent, or a special configuration.
+
+## Prerequisites
+
+*	A Virtual Machine in a VPC PrivateNetwork. Check our example below.
+
+1. When you deploy Bastion, the values are pulled from the Network in which your VM resides.
+   1. This VM doesn't become a part of the Bastion deployment itself, but you do connect to it later in the exercise.
+
+2. If you can don't have any VM connected on the private network please use the `scaleway_instance_private_nic` or the attribute `private_network` on `scaleway_instance_server` to connected to.
```
Comment (Vinzius) [inline docs/guides/using-vpc-bastion-ssh.md:20]:
```suggestion
2. If you don't have any VM connected on the private network, please use the `scaleway_instance_private_nic` or the attribute `private_network` on `scaleway_instance_server` to connect to.
```

File: docs/guides/using-vpc-bastion-ssh.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+## Config my Bastion config
+
+You should add your config on your local config file e.g: `~/.ssh/config`
+
+```shell
+Host *.myprivatenetwork
+ProxyJump bastion@<your-public-ip>:<your-port>
+```
+
+Then try to connect to. e.g:
```
Comment (Vinzius) [inline docs/guides/using-vpc-bastion-ssh.md:129]: I think the `e.g.` is implicit, so I would remove it, and add `it`
```suggestion
Then try to connect to it:
```

File: docs/guides/using-vpc-bastion-ssh.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+---
+page_title: "Using Scaleway Bastion SSH"
```
Comment (RoRoJ) [inline docs/guides/using-vpc-bastion-ssh.md:2]:
```suggestion
page_title: "Using Scaleway SSH Bastion"
```

File: docs/guides/using-vpc-bastion-ssh.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+description: |-
+  Using Scaleway Bastion SSH config.
```
Comment (RoRoJ) [inline docs/guides/using-vpc-bastion-ssh.md:4]:
```suggestion
  Using Scaleway SSH Bastion config.
```

File: docs/guides/using-vpc-bastion-ssh.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+In this guide you'll learn how to deploy Scaleway Bastion to your virtual network using the Scaleway Terraform provider.
+After Bastion is deployed, you can connect (SSH) to virtual machines in the virtual network via Bastion using the private IP address of the VM.
+When you connect to a VM, it doesn't need a public IP address, client software, agent, or a special configuration.
+
+## Prerequisites
+
+*	A Virtual Machine in a VPC PrivateNetwork. Check our example below.
+
+1. When you deploy Bastion, the values are pulled from the Network in which your VM resides.
+   1. This VM doesn't become a part of the Bastion deployment itself, but you do connect to it later in the exercise.
```
[the remark this answers is not in the document]
Comment (Monitob) [reply docs/guides/using-vpc-bastion-ssh.md:18]: This is just a Markdown style. the result is something like `1. and i.`

File: docs/guides/using-vpc-bastion-ssh.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
+---
+page_title: "Using Scaleway Bastion SSH"
+description: |-
+  Using Scaleway Bastion SSH config.
+---
+
+# How to use Scaleway VPC Bastion SSH config
+
+In this guide you'll learn how to deploy Scaleway Bastion to your virtual network using the Scaleway Terraform provider.
+After Bastion is deployed, you can connect (SSH) to virtual machines in the virtual network via Bastion using the private IP address of the VM.
```
[the remark this answers is not in the document]
Comment (Monitob) [reply docs/guides/using-vpc-bastion-ssh.md:10]: Thank you for the suggestion. I'm gonna keep your feedback.

File: docs/guides/using-vpc-bastion-ssh.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+   1. This VM doesn't become a part of the Bastion deployment itself, but you do connect to it later in the exercise.
+
+2. If you don't have any VMs connected to the Private Network, use the `scaleway_instance_private_nic` or the attribute `private_network` on `scaleway_instance_server` to connect.
+
+3. If your VMs are attached to any `scaleway_instance_ip`. You should detach them.
+
+  **Note**: You should keep your VMs and Private Network on the same zone. e.g. `fr-par-1`
+
+```hcl
+# NUMBER OF VIRTUAL MACHINES
```
Comment (remyleone) [inline docs/guides/using-vpc-bastion-ssh.md:27]: This comment line seems to be a duplicate of the description

File: docs/guides/using-vpc-bastion-ssh.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+1. When you deploy Bastion, the values are pulled from the Network in which your VM resides.
+   1. This VM doesn't become a part of the Bastion deployment itself, but you do connect to it later in the exercise.
+
+2. If you don't have any VMs connected to the Private Network, use the `scaleway_instance_private_nic` or the attribute `private_network` on `scaleway_instance_server` to connect.
+
+3. If your VMs are attached to any `scaleway_instance_ip`. You should detach them.
+
+  **Note**: You should keep your VMs and Private Network on the same zone. e.g. `fr-par-1`
+
```
Comment (remyleone) [inline docs/guides/using-vpc-bastion-ssh.md:25]: A visual representation of the architecture would be nice.

File: docs/guides/using-vpc-bastion-ssh.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+resource scaleway_vpc_public_gateway "pgw" {
+  type = "VPC-GW-S"
+  bastion_enabled = true
+  ip_id = scaleway_vpc_public_gateway_ip.pgw_ip.id
+}
+```
+
+## Configure your DHCP on your subnet
+
+To communicate over the network we require the IP address.
```
Comment (remyleone) [inline docs/guides/using-vpc-bastion-ssh.md:89]: Who is the we? Could you precise that it is the instance and they require a an IPv4 address that would be private.

File: docs/guides/using-vpc-bastion-ssh.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+```hcl
+# SCALEWAY PUBLIC GATEWAY IP
+resource scaleway_vpc_public_gateway_ip "pgw_ip" {
+}
+```
+
+## Set up your Public Gateway
+
+Public Gateways sit at the border of Private Networks and allow you to enable the bastion.
+You can also choose your port of preference on `bastion_port` option.
```
[the remark this answers is not in the document]
Comment (Codelax) [reply docs/guides/using-vpc-bastion-ssh.md:75]: It should defaults to 61000

**Review by remyleone — APPROVED** [review]

**Pull request merged**